app/guidance/page.tsx

```diff
@@ -0,0 +1,297 @@
+"use client";
+import "./guidance.scss";
+import Breadcrumbs from "@/components/Breadcrumbs";
+import Header from "@/components/Header";
+import PhaseBanner from "@/components/PhaseBanner";
+
+import SyntaxHighlighter from "react-syntax-highlighter";
+import { googlecode } from "react-syntax-highlighter/dist/esm/styles/hljs";
+
+const Guidance = ({}: {}) => {
+  const codeStyle = { backgroundColor: "transparent", margin: 0, padding: 0 };
+  return (
+    <>
+      <Header borderColour="blue-border" />
+      <div className="app-width-container">
+        <PhaseBanner
+          href="/"
+          tag={{ children: "prototype" }}
+          className="govuk-phase-banner--inverse"
+        />
+        <Breadcrumbs items={[{ text: "Home", href: "/" }]} />
+        <div className={`app-datasets`}>
+          <div className="govuk-grid-row">
+            <div className="govuk-grid-column-two-thirds">
+              <h1 className="govuk-heading-xl app-guidance__title">
+                Data catalogue API developer guidance
+              </h1>
+              <p className="govuk-body-l app-guidance__description">
+                The Data Catalogue API provides endpoints to interact with
+                datasets, editions, and versions within the IDS (Information
+                Data System). This guidance document will walk you through the
+                available endpoints and provide examples of how to use them
+                using `curl`, R, and Python.
+              </p>
+            </div>
+          </div>
+        </div>
+        <main className="app-datasets" id="main-content" role="main">
+          <div className="govuk-grid-row">
+            <div className="govuk-grid-column-two-thirds">
+              <h1 className="govuk-heading-l" id="about">
+                API Base URL
+              </h1>
+              <div className="govuk-body govuk-!-margin-bottom-0">
+                <span style={{ marginRight: 10 }}>
+                  The base URL for the Data Catalogue API is:
+                </span>
+                <code className={"app-guidance__code"}>
+                  {"https://staging.idpd.uk"}
```

Comment:
is there any way to make this dynamic, i.e whatever current domain is, otherwise we'll only end up rewriting this